.editorconfig

```diff
@@ -1,9 +0,0 @@
-root = true
```

Comment:
❓ Why did you remove this file? You don't like consistency between different IDEs?